src/VisualStudio/Core/Impl/CodeModel/FileCodeModel_Events.cs

```diff
@@ -136,7 +136,7 @@ private EnvDTE80.vsCMChangeKind ConvertToChangeKind(CodeModelEventType eventType
             return result;
         }
 
-        private void GetElementsForCodeModelEvent(CodeModelEvent codeModelEvent, out EnvDTE.CodeElement element, out object parentElement)
+        internal void GetElementsForCodeModelEvent(CodeModelEvent codeModelEvent, out EnvDTE.CodeElement element, out object parentElement)
```

Comment:
No other testcase verified this method. This was modified to be internal for the purpose of testing. Will add a comment